### ISSUE TITLE
Filter/ErrorFilter: missing error message

### ISSUE THREAD

**Issue by jcjuett** [salesforce/design-system-react#1861]
http://design-system-react-components.herokuapp.com/?selectedKind=SLDSFilter&selectedStory=ErrorFilter&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

HTML:
```
<button class="slds-button slds-button_icon slds-button_icon-bare slds-button_icon-small" title="Remove Filter: Show Me All Products" type="button">
<svg aria-hidden="true" class="slds-button__icon slds-button__icon_hint"><use xlink:href="/assets/icons/utility-sprite/svg/symbols.svg#close"></use></svg>
<span class="slds-assistive-text">Remove Filter: Show Me All Products</span>
</button>
<div>
```
Success Criteria: Error message is visible and read by screen reader

Add after <div>: 
```
<p id="sample-panel-filtering-error-error" class="slds-text-color_error slds-m-top_xx-small">Error Message</p>
```

**Note `errorMessage` is a part of the API for `FilterGroup` and not `Filter`. For now, the static HTML of an error message should be added to the Storybook Story.**

**Comment by pulkonet** [comment]
@jcjuett @interactivellama Can you explain this issue, so that it can be worked on ?

**Comment by jcjuett** [comment]
Please replicate the error field and message here: http://design-system-react-components.herokuapp.com/?selectedKind=SLDSPanel&selectedStory=Filters%20Error&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

**Comment by devgopesh** [comment]
I am working on this.

**Comment by interactivellama** [comment]
@devgopesh Thanks! It's mostly a documentation/example update.

**Comment by devgopesh** [comment]
Yes its an example update. I looked into it and we have to add "error mesage" in the error example for filter. Curently it is only showing the red outline not the text i.e. "error message"

**Comment by jcjuett** [comment]
Exactly. All you need to do is replicate the error message as it appears in
the Panel Filter Errors component.

JoAnne C. Juett. PhD
Sr. Accessibility Specialist
B2C UX Product Design & Engineering
Salesforce.com <http://salesforce.com/>
Mobile: 317.410.8784



<http://smart.salesforce.com/sig/iheyveld//us_mb/default/link.html>

**Comment by devgopesh** [comment]
@interactivellama I added the error mesage in the ErrorFilter and on npm start it gives correct output.
![Screenshot from 2019-03-29 02-00-37](https://user-images.githubusercontent.com/25543114/55190870-bbf0b680-51c6-11e9-975b-1983aeb65ea6.png)

I have wrapped the current div element inside another div and added the error message but on doing npm test it shows error in the file. Can you tell me how tabs or spaces are working in it?

**Comment by interactivellama** [comment]
This library uses story-based DOM snapshot tests built using the Jest test framework. https://github.com/salesforce/design-system-react/tree/master/tests#story-based-tests

`npm run test:snapshot -- -u` will create new snapshots. Only add the PNGs though that need updating. More than are needed might be generated.

**Comment by pulkonet** [comment]
@devgopesh Let me know if you're not working on this, I will make the changes and get it closed.

**Comment by devgopesh** [comment]
@pulkonet I am working on it.

**Comment by interactivellama** [comment]
I just added the `npm run test:snapshot:update` command if you want to test that out. Hopefully, that makes it a little more clear.
